cmd/info.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/Benchkram/errz"
+	"github.com/puppetlabs/wash/api/client"
+	cmdutil "github.com/puppetlabs/wash/cmd/util"
+	"github.com/puppetlabs/wash/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func infoCommand() *cobra.Command {
+	infoCmd := &cobra.Command{
+		Use:   "info <path>",
```

Comment:
This is mostly useful for checking what an entry's meta attribute looks like, which is useful when you're trying to figure out a meta primary query. Also open to other ways of exposing the `meta` attribute.